docs/index.rst

```diff
@@ -29,7 +29,7 @@ Content
     basics/variable_registry
     usage/style
     usage/utilities
-    usage/misc
+    usage/plot_fitting_results
```

Comment:
We can also put this under the `Documentation` chapter instead of the `Utilities`

Reply:
I think under `usage/` is still justified, because it is a specific usage that won't be used by every user.

Reply:
I meant, in which category in the screenshot do we put it?

![image](https://github.com/cyrraz/plothist/assets/43090327/447dcd78-3289-4b16-ae12-fc723130f302)
 
It is currently in `Utilities` under `misc`.

Reply:
I would keep it in utilities